src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -639,7 +642,8 @@ void OnLifeCycleEventsMovedToWindow(object? sender, EventArgs e)
 			// OnLoaded is called at the point in time where the xplat view knows it's going to be attached to the window.
 			// So this just serves as a way to queue a call on the UI Thread to see if that's enough time for the window
 			// to get attached.
-			uiView.BeginInvokeOnMainThread(() => OnLoadedCheck(null));
+			if (lifeCycleEvents is null)
+				uiView.BeginInvokeOnMainThread(() => OnLoadedCheck(null));
```

Comment:
This was only relevant when we were using Observers to try and detect loaded/unloaded changes. 
If we're wiring directly to the events then we don't need to dispatch anything